mgz/summary/map.py

```diff
@@ -148,6 +148,8 @@ def get_water_percent(tiles, dataset_id):
     """Get percent of map that is passable by ships."""
     if dataset_id not in WATER_TERRAIN:
         return None
+    if len(tiles) == 0:
```

Comment:
Tiles should never be empty. If it is, it's because of an upstream parsing problem.